sdk/boms/azure-spring-cloud-dependencies/pom.xml

```diff
@@ -44,12 +44,17 @@
     <packageOutputDirectory>${project.build.directory}</packageOutputDirectory>
     <legal><![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]></legal>
     <azure.spring.cloud.version>${project.version}</azure.spring.cloud.version>
+    <azure.resourcemanager.version>2.5.0</azure.resourcemanager.version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager;dependency} -->
     <azure.spring.identity.version>1.5.0</azure.spring.identity.version> <!-- {x-version-update;com.azure.spring:azure-identity-spring;dependency} -->
     <azure.spring.appconfigure.version>1.3.0</azure.spring.appconfigure.version> <!-- {x-version-update;com.microsoft.azure:spring-cloud-starter-azure-appconfiguration-config;dependency} -->
     <azure.core.version>1.17.0</azure.core.version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+    <azure.identity.version>1.3.1</azure.identity.version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
+    <azure.eventhub.version>5.7.1</azure.eventhub.version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs;dependency} -->
+    <azure.eventhub.checkpoint.version>1.7.1</azure.eventhub.checkpoint.version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs-checkpointstore-blob;dependency} -->
     <azure.eventhub.version>5.8.0</azure.eventhub.version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs;dependency} -->
     <azure.eventhub.checkpoint.version>1.8.0</azure.eventhub.checkpoint.version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs-checkpointstore-blob;dependency} -->
     <azure.servicebus.version>3.6.1</azure.servicebus.version> <!-- {x-version-update;com.microsoft.azure:azure-servicebus;external_dependency} -->
+    <azure.core.serializer.json.jackson.version>1.2.4</azure.core.serializer.json.jackson.version> <!-- {x-version-update;com.azure:azure-core-serializer-json-jackson;dependency} -->
```

Comment:
We have two entries here, the <azure.eventhub.checkpoint.version>

Reply:
Could you please order these properties alphabetically?

Reply:
Done